main.py

```diff
@@ -139,3 +143,5 @@ def root_login_account(_, event):
         AWSConsoleRootLogin,
     ]
 )
+
+print_rule_table(__REGISTRY)
```

Comment:
There is a `registered_rules` function you can use instead

Reply:
@maxrichie5 Done